swim/handlers_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package swim
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/uber/ringpop-go/shared"
+	"github.com/uber/ringpop-go/swim/test/mocks"
+	"github.com/uber/tchannel-go"
+)
+
+type HandlerTestSuite struct {
+	suite.Suite
+
+	testNode *testNode
+	peers    []*testNode
```

Comment:
The word peers stems more from the TChannel lexicon than the Ringpop. It's rarely used, if ever, in ringpop-node. And that's intentional so as not to make the concept of peers in TChannel the same as Ringpop. To be consistent with history and ringpop-node, can you call rename peers to nodes or make `cluster` itself destroyable?

That said, `peers` does not appear to be used for anything other than to contain `testNode`. Can we get rid of `peers` altogether?